Esri/arcgis-maps-sdk-swift-samples
# [Update] Add done buttons to popovers

## Description

This PR adds "Done" buttons to the `popovers` that were missing them. This allows users to close them when in landscape mode.

## Linked Issue(s)

- `swift/issues/5233`

## How To Test

- Ensure the done buttons and sheets work in modified samples on iPhone, iPad, and Mac Catalyst.

<details><summary>Helper Testing Code</summary><p>

```swift
/// This property helps test the samples affected by this PR.  To use it, replace the
/// `FavoritesView.favoriteSamples` property in `FavoritesView.swift` with the following
/// code. The samples needing testing will then show up under the "Favorites" category when the app is run.
private var favoriteSamples: [Sample] {
    let sampleViews = [
        ChangeMapViewBackgroundView(),
        DensifyAndGeneralizeGeometryView(),
        ManageBookmarksView(),
        ManageOperationalLayersView(),
        SetVisibilityOfSubtypeSublayerView(),
        ShowViewshedFromPointInSceneView()
    ] as [Any]
    
    return sampleViews.compactMap { view in
        let viewName = String(describing: view).components(separatedBy: "(").first
        return SamplesApp.samples.first { sample in
            sample.nameInUpperCamelCase.appending("View") == viewName
        }
    }
}
```

</p></details>
